packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -292,7 +300,7 @@ const getStyles = (theme: GrafanaTheme2) => {
     content: css({
       label: 'panel-content',
       flexGrow: 1,
-      contain: 'strict',
+      contain: height ? 'strict' : 'none',
```

Comment:
I've tested passing 'content' when height is defined, but in this case the sticky search bar gets pushed down under the title header, like this:

![Screenshot 2023-06-23 at 10 21 10 AM](https://github.com/grafana/grafana/assets/58232930/081b4ba5-b91d-4ac9-a337-5d0bcf28a446)

I think it might be a good to just set it to 'none' when height is not passed